App/server/tests/test_scheduled_actuator.py

```diff
@@ -0,0 +1,113 @@
+import os
+import sys
+
+sys.path.append("../server")
+
+# sys.path.append('..')
+
+# from dotenv import dotenv_values
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from pymongo import MongoClient
+
+parent = os.path.abspath(".")
+sys.path.append(parent)
+
+
+print(sys.path)
+
+from App.server.routes.scheduled_actuator import router as sa_router
+
+app = FastAPI()
+# config = dotenv_values(".env")
+app.include_router(sa_router, tags=["scheduled_actuators"], prefix="/sa")
+
+
+@app.on_event("startup")
+async def startup_event():
+    app.mongodb_client = MongoClient(os.environ["ATLAS_URI"])
+    app.database = app.mongodb_client[os.environ["DB_NAME"] + "test"]
+
+
+@app.on_event("shutdown")
+async def shutdown_event():
+    app.mongodb_client.close()
+    app.database.drop_collection("scheduled_actuators")
```

Comment:
this should be "scheduled actuators" to match the collection referenced in the routes (the database["scheduled actuators"]  in scheduled_actuators.py)